docs/roadmap/cosmos-hub-roadmap-2.0.md

```diff
@@ -1,66 +1,28 @@
 # The Cosmos Hub Roadmap
 
-This Cosmos Hub roadmap serves as a reference for the current planned features of upcoming releases, as well as providing a record of past releases.
+This Cosmos Hub roadmap serves as a reference for the current planned features of upcoming releases. For past releases, please see the following [document](./README.md).
```

Comment:
@jtremback pulled this content from the January blog. Can add the upcoming releases that we have planned (10 & 11)

Reply:
Now use the living roadmap google doc link that wasn't available before.